native/cocos/renderer/gfx-wgpu/WGPUExports.h

```diff
@@ -1,4 +1,10 @@
 #pragma once
+#include <boost/fusion/include/adapt_struct.hpp>
+#include <boost/fusion/include/at_c.hpp>
+#include <boost/fusion/include/for_each.hpp>
```

Comment:
boost::fusion might greatly increase compilation time. Is it necessary in header file?